packages/gatsby-source-wordpress/src/normalizer.js

```diff
@@ -0,0 +1,65 @@
+const _ = require(`lodash`)
+
+class Normalizer {
+
+    /**
+     * Normalizer Constructor
+     * 
+     * @param  {Array} entitys
+     * @param  {Object} args
+     * @return {Object} this
+     */
+    constructor(entitys, args) {
+        this.entitys = entitys
+        this.args = args
+        this.normalizers = []
+        this.queue = []
+    }
+
+    /**
+     * Set Normalizers
+     * 
+     * @param {Function} normalizer
+     * @param {null|Number} priority
+     * @return {Object}
+     */
+    set(normalizer, priority = null) {
+        let property = priority === null ? `queue` : `normalizers`
+
+        this[property].push({
+            normalizer,
+            priority,
+        })
+
+        return this
+    }
+
+    /**
+     * Normalize the entities
+     * 
+     * @return {Array}
+     */
+    async normalize() {
+        var normalizers = this.getNormalizers()
+        for (let i = 0; i < normalizers.length; i++) {
+            this.entitys = await normalizers[i].normalizer(this.entitys, this.args)
+        }
+
+        return this.entitys
+    }
+
+    /**
+     * Concat the queue and prioritised normalizers
+     * 
+     * @return {Array}
+     */
+    getNormalizers() {
+        return _.concat(
```

Comment:
Use `Array.concat()` instead of lodash eliminates the need for extra dependency call in this file.